azimuth_shr/loading_resources.py

```diff
@@ -67,3 +67,32 @@ def load_CLINC150_data(full_path, python_loader, train=True) -> Dataset:
     if not train:
         dst.pop("train")
     return dst
+
+
+def align_labels(ds_dict: DatasetDict, azimuth_config: AzimuthConfig) -> DatasetDict:
+    features: Features = [v.features for v in ds_dict.values()][0]
+    if not isinstance(features[azimuth_config.columns.label], ClassLabel):
+        # Get all classes from both set and apply the same mapping to every dataset.
+        classes = sorted(
+            list(set(sum([ds[azimuth_config.columns.label] for ds in ds_dict.values()], [])))
```

Comment:
Again, not new code, but this way of concatenating lists with `sum([...], [])` is not super readable and it's apparently very inefficient (I say this article https://mathieularose.com/how-not-to-flatten-a-list-of-lists-in-python).
Instead, we can directly "flatten" the lists in the set comprehension:
```suggestion
            list(set(label for ds in ds_dict.values() for label in ds[azimuth_config.columns.label]))
```

Reply:
Thank you, yeah the `sum` was puzzling me!